packages/dds/tree/src/feature-libraries/defaultFieldKinds.ts

```diff
@@ -113,22 +118,25 @@ function replaceRebaser<T>(data: {
             }
             return { old: f[0].old, new: f[f.length - 1].new };
         },
-        invert: (changes: ReplaceOp<T>) => changes === 0 ? 0 : { old: changes.new, new: changes.old },
-        rebaseAnchors: data.rebaseAnchors,
+        invert: (changes: ReplaceOp<T>, invertChild: NodeChangeInverter) => {
+            return changes === 0 ? 0 : { old: changes.new, new: changes.old };
+        },
```

Comment:
Is there a reason why this can't remain
```suggestion
        invert: (changes: ReplaceOp<T>, invertChild: NodeChangeInverter) => changes === 0 ? 0 : { old: changes.new, new: changes.old },
```

Reply:
Are you asking if it could still be one line? Your suggestion is longer than 120 characters.